### ISSUE TITLE
bug/file support for plain text

### ISSUE THREAD

**Issue by younes-io** [Unstructured-IO/unstructured#2628]
**Describe the bug**
I have .txt files that contain legal contents and when I process them, I get an error `The MIME type of 'processed_documents/law.txt' is 'message/news'. This file type is not currently supported in unstructured.`

**To Reproduce**
Just load the .txt files like this:

```python
loader = DirectoryLoader(path='./processed_documents/', glob="**/*.txt")

docs = loader.load()
```

**Expected behavior**
It should be processed as a plain text!! Why should contents be the reason of rejection? (if I change the contents to something not related to law different it gets processed).. If I had a few files it's okay, but I have hundreds !!

**Comment by scanny** [comment]
@younes-io can you post a file that gives you this error? `.txt` files are partitioning just fine for me, I expect there is some artifact in the file contents that is triggering recognition of the file type as `message/news`.

Does the file perhaps have the string: "Path:" or "Xref:" toward the top? Could these text files possibly have been downloaded from a usenet newsgroup?

If you know all these files are plain text you can add a `loader_kwargs={"content_type": "text/plain"}` argument and that will instruct `unstructured` to skip the file-type detection step.

**Comment by younes-io** [comment]
@scanny: Unfortunately, I can't share the files. I created  the files myself, and they just contain legal clauses (human language, no path, no xref).
I switched to another library for file loading. Unstructured is unstable on this part. I can't tell users we'll process their files and then eventually not process them because unstructured doesn't support a plain text file type.

Thank you though for your tips
